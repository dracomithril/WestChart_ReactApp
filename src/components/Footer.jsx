/**
 * Created by Gryzli on 12.02.2017.
 */
import React from 'react';

export default class Footer extends React.Component {
    render() {
        return (<footer className="footer">
                    <span >{'site created by '}<a
                        href="https://github.com/dracomithril">dracomithril</a>{' v'+process.env.npm_package_version+' Â© Copyright 2017'}</span><br/>
            <span>{'Any questions? '}<a
                href="mailto:dracomithril@gmail.com?subject=[WCSChartAdmin]">contact me</a></span>
        </footer>);
    }
}
Footer.propTypes = {};