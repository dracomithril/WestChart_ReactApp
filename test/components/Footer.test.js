/**
 * Created by XKTR67 on 4/19/2017.
 */
import React from 'react';
import PropTypes from 'prop-types';
import Footer from './../../src/components/Footer';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import configureMockStore from 'redux-mock-store';

const mockStore = configureMockStore([]);
describe('<Footer/>', () => {
    it('renders without crashing', () => {
        const store = mockStore({});
        const wrapper = shallow(
            <Footer />, {
                context: {store},
                childContextTypes: {store: PropTypes.object}
            }
        );
        expect(shallowToJson(wrapper)).toMatchSnapshot();

    });
});
