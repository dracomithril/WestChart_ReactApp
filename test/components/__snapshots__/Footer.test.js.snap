// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> renders without crashing 1`] = `
<footer
  className="footer"
>
  <span>
    site created by 
    <a
      href="https://github.com/dracomithril"
      target="_newtab"
    >
      dracomithril 
    </a>
    repo on
    <a
      href="https://github.com/dracomithril/WestChart_ReactApp"
      target="_newtab"
    >
       github
      <i
        aria-hidden="true"
        className="fa fa-github"
      />
    </a>
     
     v0.6.3 Â© Copyright 2017
  </span>
  <br />
  <span>
    Any questions? 
    <a
      href="mailto:dracomithril@gmail.com?subject=[WCSChartAdmin]"
    >
      contact me
    </a>
  </span>
  <a
    href="/api/fb_policy"
  >
     Facebook Policy
  </a>
</footer>
`;
